docs/docs/rest-api/public/api/v2/types/queue.raml

```diff
@@ -41,7 +52,7 @@ types:
         type: datetime
         description: Point in time when the last used offer has been processed.
       rejectSummaryLastOffers?:
```

Comment:
Also compute rejectSummaryLaunchAttempt the same way as  rejectSummaryLastOffers